sci-mathematics/sage/sage-6.8-r1.ebuild

```diff
@@ -61,6 +61,7 @@ CDEPEND="dev-libs/gmp:0=
 	>=sci-mathematics/eclib-20150510[flint]
 	>=sci-mathematics/gmp-ecm-6.4.4[-openmp]
 	>=sci-mathematics/flint-2.4.4[ntl]
+	<sci-mathematics/flint-2.5[ntl]
 	~sci-libs/fplll-4.0.4
```

Comment:
To be honest I would just drop old revisions. Since discovering the double constraint problem I have mostly relied on permissive constraint and keywording - of course that doesn't work well on `~arch`. I have considered your solution above but I really try to avoid it.